uk.co.compucorp.civicrm.hrleaveandabsences/sql/auto_install.sql

```diff
@@ -142,4 +142,80 @@ CREATE TABLE `civicrm_hrleaveandabsences_notification_receiver` (
       FOREIGN KEY (`type_id`) REFERENCES `civicrm_hrleaveandabsences_absence_type`(`id`) ON DELETE CASCADE,
     CONSTRAINT FK_civicrm_hrleaveandabsences_notification_receiver_contact_id
       FOREIGN KEY (`contact_id`) REFERENCES `civicrm_contact`(`id`) ON DELETE CASCADE
+)  ENGINE=InnoDB DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci  ;
+
+-- /*******************************************************
+-- *
+-- * civicrm_hrleaveandabsences_work_pattern
+-- *
+-- * This entity holds the basic description about a Work Pattern
+-- *
+-- *******************************************************/
+CREATE TABLE `civicrm_hrleaveandabsences_work_pattern` (
```

Comment:
These are also automatically generate by GenCode. But in this case, since we have to manually copy the SQL statements from civicrm.mysql, I think there's no problem to properly format them.